baseapp/params.go

```diff
@@ -84,3 +85,14 @@ func ValidateValidatorParams(i interface{}) error {
 
 	return nil
 }
+
+// ValidateVersionParams defines a stateless validation on VersionParams. This
+// function is called whenever the parameters are updated or stored.
+func ValidateVersionParams(i interface{}) error {
+	_, ok := i.(tmproto.VersionParams)
```

Comment:
Just a small doubt, why are we ignoring the returned value here, don't we need to validate it ?
Something like 
```suggestion
	v, ok := i.(tmproto.VersionParams)
	if !(v.AppVersion > 0) {
		return errors.New("updated app version must be greater than zero")
	}
```
please clarify me if I'm wrong

Reply:
The app could say the version is 0. outside it being a number it can be anything.